ui/org.eclipse.pde.ui/src/org/eclipse/pde/ui/launcher/PluginsTab.java

```diff
@@ -217,16 +240,9 @@ public Image getImage() {
 		return fImage;
 	}
 
-	/**
-	 * Validates the tab.  If the feature option is chosen, and the workspace is not correctly set up,
-	 * the error message is set.
-	 *
-	 * @see org.eclipse.pde.ui.launcher.AbstractLauncherTab#validateTab()
-	 */
 	@Override
 	public void validateTab() {
-		String errorMessage = null;
-		setErrorMessage(errorMessage);
+		setErrorMessage(null);
```

Comment:
This looks suspicious (even if not introduced by you), at least some comment is needed to explain why we don't set the error message.

Once again: this change has no relationship to the fix at all, it is a cleanup and ideally should be done before/after the actual fix.

Reply:
Moved to https://github.com/eclipse-pde/eclipse.pde/pull/1259